tests/file_storage/tests.py

```diff
@@ -944,6 +944,18 @@ def test_filefield_default(self):
         self.assertEqual(obj.default.read(), b"default content")
         obj.default.close()
 
+    def test_filefield_db_default(self):
+        temp_storage.save("tests/db_default.txt", ContentFile("default content"))
+        obj = Storage.objects.create()
+        self.assertEqual(obj.db_default.name, "tests/db_default.txt")
+        self.assertEqual(obj.db_default.read(), b"default content")
+        obj.db_default.close()
+
+        obj.delete()
+        obj = Storage()
+        self.assertEqual(obj.db_default.read(), b"default content")
```

Comment:
I'm missing why we are not testing the `name` attr here :thinking:

Reply:
It was copying the test from above but I've added this in now